ui-src/src/app/configurations/configurations.component.ts

```diff
@@ -83,7 +87,9 @@ export class ConfigurationsComponent implements OnInit {
       "flex-flow": "row wrap",
```

Comment:
Is this the layout for the auto generated input box's?

Reply:
ya this to used to put the boxes in a row

Reply:
Can we put this in a separate file.

Reply:
this is done in story 46, will i merge the two stories together together?

Reply:
no its ok, as long as it is done.